README.md

```diff
@@ -8,6 +8,7 @@ A VLC-powered alternative to Flutter's video_player that supports iOS and Androi
 
 <br>
 
+[![Join the chat at https://discord.gg/3h3K3JF](https://img.shields.io/discord/716939396464508958?label=discord)](https://discord.gg/3h3K3JF)
```

Comment:
```suggestion
[![Join the chat at https://discord.gg/mNY4fjVk](https://img.shields.io/discord/716939396464508958?label=discord)](https://discord.gg/mNY4fjVk)
```
Should we point directly to Flutter channel then?

Reply:
didn't know that was an option, sure go for it!